x-pack/plugins/alerting/server/rules_client/lib/should_skip_attributes_update.test.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { BulkEditSkipReason } from '@kbn/security-solution-plugin/common/detection_engine/rule_management/api/rules/bulk_actions/response_schema';
+import { BulkEditOperation } from '../..';
+import { RawRule } from '../../types';
+import { shouldSkipAttributesUpdate } from './should_skip_attributes_update';
+
+describe('shouldSkipAttributesUpdate', () => {
```

Comment:
I think instead of covering this small helper function we should add more tests to `x-pack/plugins/alerting/server/rules_client/tests/bulk_edit.test.ts` that would cover the whole behavior of skipping rule edits. These would be much more stable tests.

Reply:
Covered behaviour with tests in `x-pack/plugins/alerting/server/rules_client/tests/bulk_edit.test.ts